### ISSUE TITLE
quickey accessibility issues

### ISSUE THREAD

**Issue by yplassiard** [fwextensions/QuicKey#131]
Found some accessibility issues while using the app:
- when searching for elements, the results are not accessible to a screen reader.
- enh: could be nice to select results using arrow keys after typing some letters, then return to activate.
